src/ios/CDVCamera.m

```diff
@@ -58,6 +59,30 @@ Licensed to the Apache Software Foundation (ASF) under one
     }
 }
 
+NSString* ExtensionFromCDVEncodingType(CDVEncodingType encoding) {
+    switch (encoding) {
+        case EncodingTypeJPEG:
+            return @"jpg";
+        case EncodingTypePNG:
+            return @"png";
```

Comment:
Do we need also to handle the heic format?

Reply:
Those are the target encoding types. The Plugin doesn't support others so far. I just wanted to get rid of an if else for enum case.